src/main/config/config.c

```diff
@@ -832,6 +832,10 @@ void activateConfig(void)
 
 void validateAndFixConfig(void)
 {
+    if (config->motor_pwm_protocol = PWM_TYPE_BRUSHED && escAndServoConfig->mincommand < 1000) {
```

Comment:
I'm actually out and this laptop has only notepad. Probably I shall wait till I get home to do this properly. Sorry for all the inconvenience caused.